### PR TITLE
Prepare release 0.98.0

### PR DESCRIPTION
**Pull request by alejoe91**

### CODE REVIEW

File: doc/releases/0.98.0.rst [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+preprocessing:
+  * Add IBL channel interpolation to preprocessing (#1025)
+  * Add IBL high pass spatial filter to preprocessing (#1059)
+  * Add IBL bad channel detection (#1070)
+  * Add get_spatial_interpolation_kernel in preprocessing (#1074)
+  * Add median/average artefact modes to :code:`remove_artifacts` (#1106, #1143, #1250)
+  * Fix phase-shift dtype (#1172)
+  * Add correct_lsb function (#1262)
+
```
Comment (h-mayorquin) [inline doc/releases/0.98.0.rst:79]: Maybe the padding and the dtype here?

Comment (alejoe91) [reply doc/releases/0.98.0.rst:79]: I forgot to save the file! It was supposed to be a blank file eheheh

Comment (alejoe91) [reply doc/releases/0.98.0.rst:79]: I'll start with the proper release notes tomorrow

File: doc/releases/0.98.0.rst [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+7th July 2023
+
+
+Main changes:
+
+* Implemented motion correction in SpikeInterface
+* Get rid of to_dict in _kwargs and use JSON serializer
+* _ind to _index in complex dtype representation
+* Moved to src/spikeinterface structure
+* Reformat to black and use pre-commit CI
```
Comment (samuelgarcia) [inline doc/releases/0.98.0.rst:15]: I do not feel that black and pre-commit is somhow an improvement...
Promoting to *main changes* hurt me a lot.

**Comment by samuelgarcia** [comment]
Do not forget to also siwtch to a stable neo/probeinterface in pyproject.toml

**Review by samuelgarcia — APPROVED** [review]

**Pull request merged**